pkg/kubectl/configmap.go

```diff
@@ -90,8 +85,26 @@ func (s ConfigMapGeneratorV1) Generate(genericParams map[string]interface{}) (ru
 		delegate.EnvFileSource = fromEnvFile
 		delete(genericParams, "from-env-file")
 	}
+	hashParam, found := genericParams["hash"]
+	if found {
+		hashBool, isBool := hashParam.(bool)
```

Comment:
While it's true that in this case, since `delegate.Hash` would already have the zero value (`delegate := &ConfigMapGeneratorV1{}`), changing to `delegate.Hash, found := genericParams["hash"]` wouldn't risk [unsetting](https://golang.org/ref/spec#Type_assertions) `delegate.Hash`, I think the pattern of only modifying when the param is found is safer in general. And I'd like to maintain consistency with the rest of the file :).